python/cudf/cudf/core/index.py

```diff
@@ -2329,6 +2329,7 @@ def __init__(
             data = column.as_column(data)
             data.dtype.closed = closed
 
+        self.closed = closed
```

Comment:
Was this just a bug before? I see that `cut` at least always expected this attribute to exist.

Reply:
I think so..